<?xml version="1.0"?>
<robot name="cyan_block">
	<link name="base_link">
		<visual>
			<geometry>
				<mesh filename="package://irsl_sim_environments/urdf/puzzle_blocks/cyan_block/cyan_block_base_link.dae" />
			</geometry>
		</visual>
		<collision>
			<origin xyz="0.000000000000e+00 0.000000000000e+00 0.000000000000e+00" rpy="0.000000000000e+00 -0.000000000000e+00 0.000000000000e+00" />
			<geometry>
				<box size="3.000000000000e-02 3.000000000000e-02 3.000000000000e-02" />
			</geometry>
		</collision>
		<collision>
			<origin xyz="0.000000000000e+00 -3.000000000000e-02 3.000000000000e-02" rpy="0.000000000000e+00 -0.000000000000e+00 0.000000000000e+00" />
			<geometry>
				<box size="3.000000000000e-02 3.000000000000e-02 3.000000000000e-02" />
			</geometry>
		</collision>
		<collision>
			<origin xyz="0.000000000000e+00 0.000000000000e+00 3.000000000000e-02" rpy="0.000000000000e+00 -0.000000000000e+00 0.000000000000e+00" />
			<geometry>
				<box size="3.000000000000e-02 3.000000000000e-02 3.000000000000e-02" />
			</geometry>
		</collision>
		<collision>
			<origin xyz="0.000000000000e+00 3.000000000000e-02 3.000000000000e-02" rpy="0.000000000000e+00 -0.000000000000e+00 0.000000000000e+00" />
			<geometry>
				<box size="3.000000000000e-02 3.000000000000e-02 3.000000000000e-02" />
			</geometry>
		</collision>
		<inertial>
			<mass value="0.021599999999999998" />
			<origin xyz="0.000000000000e+00 0.000000000000e+00 2.250000000000e-02" />
			<inertia ixx="1.6604999999999998e-05" ixy="0" ixz="0" iyy="6.8849999999999976e-06" iyz="0" izz="1.2959999999999998e-05" />
		</inertial>
	</link>
<!-- HotFix for gazebo (manually added) -->
<gazebo reference="base_link">
 <collision>
  <surface>
   <contact>
    <ode>
     <kp>500.0</kp>
     <kd>0.01</kd>
     <max_vel>0.0</max_vel>
     <min_depth>0.0004</min_depth>
    </ode>
   </contact>
   <friction>
    <ode>
     <mu>10.0</mu>
     <mu2>10.0</mu2>
    </ode>
   </friction>
  </surface>
 </collision>
</gazebo>
</robot>
